style.css

```diff
@@ -6,23 +6,22 @@ form {
     background-color: skyblue;
     margin: auto;
     opacity: 0.8;
-    min-height: 300px;
+    min-height: 370px;
 }
 h1 {
     text-align: center;
     font-size: 40px;
     border-bottom: 3px solid black;
 }
-.name-style {
+.caption {
     font-size: 20px;
+    margin-left: 10px;
 }
-.nick-style {
-    font-size: 20px;
-}
-.your-name {
+.text-window {
     font-size: 15px;
+    margin-left: 10px;
 }
-.your-nick{
+.choose {
```

Comment:
it is good to name css classes using nouns (rzeczownik in Polish). So - please change to `choice`. Hovewer this class names changes are super great! good job 🎉 You implemented first basic design patter (very helpful rule for programmers - DRY: dont repeat yourself).